Wrappers/Python/cil/processors/CofR_xcorrelation.py

```diff
@@ -109,11 +109,23 @@ def process(self, out=None):
 
             target -= 360     
 
-        ind = np.abs(angles_deg - target).argmin()
-        
-        ang_diff = abs(angles_deg[ind] - angles_deg[0])
+        ind1 = np.abs(angles_deg - target).argmin()
+        ang_diff1 = abs(angles_deg[ind1] - angles_deg[0])
+
+        target2 = target+360
+        ind2 = np.abs(angles_deg - target2).argmin() 
+        ang_diff2 = abs(angles_deg[ind2] - angles_deg[0])
+
+        if abs(ang_diff1-180)>abs(ang_diff2-180):
+            ind = ind2
+            ang_diff = ang_diff2
+            target = target2
+        else:
+            ind = ind1
+            ang_diff = ang_diff1
```

Comment:
Instead if having this check as well as the if statement further up establishing the target value, can we combine them or replace them with a single check to establish which target should be used?

Reply:
Hi Laura, thank you for reviewing :) I've updated this now so I think it can use just one check for both combinations of angles